### DISCUSSION TITLE
Ability for the administrator to create different types of additional user fields

### DISCUSSION THREAD

**Discussion by alex7070m** [pH7Software/pH7-Social-Dating-CMS#1075] [Ideas/Suggestions]
Hello!
Now it is possible to create only 2 types of additional fields: Text Box and Number. It would be nice to be able to set different types of fields: Drop-down list, Radio buttons, Checkboxes .  For example this is necessary so that the user can choose an attitude towards bad habits, the choice of education, religion, and more.

**Comment by rthuey2** [comment]
currently trying to figure this out. can see where it can be done in the database. not sure how to make it reflect on the webpage
